### ISSUE TITLE
Screen Reader: Landing Page: The logo image of Expensify is not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76936]
## Action Performed:
1. Using JAWS + Chrome, open the page https://new.expensify.com/
2. Press the arrow keys and try to focus the 'Expensify' logo icon at the top, left corner of the screen.

## Expected Result:
The informative Expensify logo will be focused and announced to users, e.g.:
- 'Expensify'

## Actual Result:
The logo image of Expensify is not announced.

HTML:
<svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 78 19" width="100%" height="100%" hovered="false" pressed="false"><path d="..." style="fill: rgb(3, 212, 124);"></path></svg>

## Area issue was found in:
Landing Page

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Give the logo role="img" and add an aria-label or <title> that matches the logo name, e.g.:
- 'Expensify'
https://www.magentaa11y.com/#/web-criteria/component/informative-image?tab=2

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/7cc06ef9-e1bc-4041-886a-69356d1688b0

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995078
**Issue reported by:** NA
**Slack conversation:** NA
